.travis.yml

```diff
@@ -1,15 +1,28 @@
 language: android
 jdk: oraclejdk8
-dist: trusty
-sudo: true
+#dist: trusty
+#sudo: true
```

Comment:
I believe sudo is recommended when using emulator because you get almost double the RAM and more than double the disk space

https://docs.travis-ci.com/user/reference/overview/#virtualization-environments